articles/express/index.md

```diff
@@ -0,0 +1,360 @@
+```
+
+Let's create a basic Express example app.
+
+```JavaScript
+//app.js
+var express = require("express");
+var app = express();
+
+//2
+app.get("/", function(req, res){
+  res.send("HELLO WORLD");
+  });
+
+//3
+app.listen(3000, function(){
+  console.log("Application started and Listening on port 3000");
+  });
+```
+
+The code above creates a basic Express application. To run this script, go to your command prompt and enter the command `node app.js` in the project directory. 
+
+In the console, we can see `Application started and Listening on port 3000` and if we visit `http://localhost:3000/` we can see `HELLO WORLD`.
+
+Let's look at what the code above does.
+
+The first line imports the express module. The second line creates an Express application by calling the top-level `express()` function. 
+
+Our `app` variable (express application) has methods for handling requests and configuring how the application behaves. We can create multiple apps this way, each with their own requests and responses.
+
+Lets examine the code in section two. `app.get()` is a function, called *route definition*, which tells the express app how to handle an HTTP `GET` request to our server. 
+
+This function takes two main parameters, the first is the route or path which is the relative path from the root of the server; the second is a function that is invoked whenever there is a request to that path. 
+
+In this case, we are listening for `GET` requests to `/` which is the root of the website.
+
+The second parameter, the callback function, has two arguments `req` and `res`. `req` represents the **request** sent from the browser to the server. `res` represents the **response** that the server sends back.
+
+The code in section three starts a server on the port 3000. You can go to `localhost:3000` to view your response.
+
+### Core parts of Express
+
+#### Middleware
+
+Middleware is a set of functions that sit between a raw request and the final intended route. Middleware functions have access to *all* the HTTP requests coming to the server. Middleware can handle tasks such as logging, sending static files, authorization and session management, etc.  
+
+In Node.js, the request and response objects are passed to one function (request handler) that we write, in Express these objects are passed through a set of functions, called the **middleware stack**. 
+
+Express will start at the first function in the stack and execute in order down the stack.
+
+Every function in the stack takes three arguments `request`, `response` and `next`. `next` is a function, that when called Express executes the next function in the stack. This is a subtle difference between middleware and a route handler which we saw above.
+
+Let's look at a basic static file server to understand middleware. Initialize an npm project and install express. Create a directory named `static` and copy-paste *any* available static files into the folder (text, images, etc.).
```

Comment:
You've already told readers to install Express earlier in the article. Best practise would be to continue on from the first code snippet.